torch/_torch_docs.py

```diff
@@ -152,6 +152,34 @@ def merge_dicts(*dicts):
     tensor([ 1.2294,  2.2004,  1.3690,  1.7298])
 """.format(**common_args))
 
+add_docstr(torch.acosh,
+           r"""
+acosh(input, out=None) -> Tensor
+
+Returns a new tensor with the inverse hyperbolic cosine of the elements of :attr:`input`.
+The domain of the inverse hyperbolic cosine is `[1, inf)` and values outside this range
+will be mapped to ``NaN``, except for `+ INF` for which the output is mapped to `+ INF`.
+
+.. math::
+    \text{out}_{i} = \cosh^{-1}(\text{input}_{i})
+""" + r"""
+Args:
+    input (Tensor): the input tensor where :math:`input` is in the interval `[1, inf)`
```

Comment:
Since this is now redundant with the explanation, actually seems OK to just delete the comment (analogous to add). Sorry for the confusion.

Reply:
Agreed, this makes it much simpler. :) Thanks!